### ISSUE TITLE
SignalRGB problems

### ISSUE THREAD

**Issue by joshuadyson** [fu-raz/FanControlThermaltake#8]
Thermaltake hubs toggled on in SignalRGB, the fans stop responding to can control settings, but the moment i toggle them off in signal RGB, they start working again instantly. 
when the setting is toggled on the rpms will lock on what ever rpm it was on last. 
![Screenshot_751](https://user-images.githubusercontent.com/55064663/236161171-f7438e7a-a81d-4bee-8321-cf85fd8c0d85.png)

**Comment by fu-raz** [comment]
I think the problem might be that the TT device only supports having one connection? Maybe it will only accept commands from one app? And if you toggle on the device in signal RGB, it takes control and Fancontrol loses control?
I'll see if I can ask the SRGB devs

**Comment by fu-raz** [comment]
Yup, problem is that there can only be one app controlling the device
